os-autoinst-openvswitch

```diff
@@ -33,6 +37,10 @@ sub init_switch {
     die "can't parse bridge local port MAC" unless $self->{MAC};
     die "can't parse bridge local port IP"  unless $self->{IP};
 
+    ## coolo and richard terrible hack to make tap devices
+    for my $i (0..10) {
+      system("ovs-vsctl add-port $self->{BRIDGE} tap$i tag=999");
+    }
 
```

Comment:
indeed - teamd suite fails right now as it expects 133.

This PR is mainly to document what we applied - it feels rather dirty